modules/oadp-configuring-dpa-multiple-bsl.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/configuring-oadp-multiple-bsl.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-configuring-dpa-multiple-bsl_{context}"]
+= Configuring the DPA with more than one BSL
+
+You configure the DPA with more than one BSL and specify the credentials provided by the cloud provider.  
+
+.Prerequisites
+
+* You must install the {oadp-short} Operator.
+* You must create the secrets by using the credentials provided by the cloud provider.
+
+.Procedure
+
+. Configure the DPA with more than one BSL. See the following example.
++
+.Example DPA
+[source, yaml]
+----
+#...
+backupLocations:
+  - name: aws # <1>
+    velero:
+      provider: aws
+      default: true
+      objectStorage:
+        bucket: <bucket_name> # <2>
+        prefix: <prefix> # <3>
+      config:
+        region: <region> # <4>
+        profile: "default"
+      credential:
+        key: cloud
+        name: cloud-credentials # <5>
+  - name: odf # <6>
+    velero:
+      provider: aws
+      default: false
+      objectStorage:
+        bucket: <bucket_name> 
+        prefix: <prefix> 
+      config:
+        profile: "default" 
+        region: <region_name>
+        s3Url: <url> # <7>
+        insecureSkipTLSVerify: "true"
+        s3ForcePathStyle: "true"
+      credential:
+        key: cloud
+        name: <custom_secret_name_odf> # <8>
+#...
+----
+<1> Specify a name for the first BSL.
+<2> Specify a bucket name.
```

Comment:
instead of "a" I would use "the" because the bucket must be already be created prior to this step. Using "a" can give the wrong impression it will be created for you

Reply:
done.